### DISCUSSION TITLE
Arm Docker Image ?

### DISCUSSION THREAD

**Discussion by geminigeek** [cloudamqp/lavinmq#369] [Ideas]
hi,

just started playing with lavinmq, seems an excellent product, can we have an arm based docker image to run on raspberry pi?

**Comment by carlhoerberg** [comment]
Yes, of course: https://hub.docker.com/r/cloudamqp/lavinmq/tags arm64 only though, so rpi3 and rpi4 only.

**Reply by geminigeek** [reply]
thanks, just tested it on pi4 working gr8
